src/analytics/lambdas/load-data-workflow/create-load-manifest.ts

```diff
@@ -160,9 +160,15 @@ export const handler = async (_event: any, context: Context) => {
     };
   }
 
+  const response = await doManifestFiles(candidateItems, queryResultLimit, processJobNum, tableName, requestId);
```

Comment:
there is some conflicts with latst main, let's resolve them locally

Reply:
done